netbox/virtualization/models/clusters.py

```diff
@@ -107,8 +118,8 @@ class Meta:
                 name='%(app_label)s_%(class)s_unique_group_name'
             ),
             models.UniqueConstraint(
-                fields=('site', 'name'),
-                name='%(app_label)s_%(class)s_unique_site_name'
+                fields=('_site', 'name'),
+                name='%(app_label)s_%(class)s_unique__site_name'
             ),
```

Comment:
I think this still works as we set all the underscored scope vars if a higher one is set in the chain region -> site group -> site -> location.  So if location is set then _site is also set correctly.  This would make the name only needing to unique to a Site and not a region, which seems to be still valid?  Or is it just un-necessary now?

Reply:
I was thinking more re: location assignment, but I think you're right. We can leave it as-is.